mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -24,7 +25,7 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 	FIELD field_8037 EMPTY Lnet/minecraft/class_1799;
 	FIELD field_8038 item Lnet/minecraft/class_1792;
 	FIELD field_8039 lastDestroyPos Lnet/minecraft/class_2694;
-	FIELD field_8040 tag Lnet/minecraft/class_2487;
+	FIELD field_8040 nbt Lnet/minecraft/class_2487;
```

Comment:
If this rename is made, should document that the nbt is stored in `Tag` sub compound within the nbt form of an item stack.

Also, doesn't mojang have a constant for `Tag` when they have those for `CanDestroy` etc?

Reply:
Mojang doesn't seem to have a constant for the `tag` key in the serialized NBT.